privacy-protections/request-blocklist/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Request Blocklist test page</title>
+
+  <style>
+    #notes { margin-bottom: 20px; }
+    #notes > * { margin: 0; font-style: italic; }
+    table { border-collapse: collapse; }
+    td { margin: 0; padding: 0px; width: 200px; height: 100px; border: 1px #000 solid; }
+    img, iframe { display: block; width: 100%; height: 100%; margin: 0; padding: 0; border: 0; }
+    .allow { background-color: red; background-image: url("./blocked-fail.png"); }
+    .block { background-color: green; background-image: url("./blocked-pass.png");}
+  </style>
+</head>
+<body>
+  <p><a href="../index.html">[Home]</a></p>
+
+  <p>
+    Simple test page that serves as a quick visual test to show if the Request
+    Blocklist feature is active and working.
```

Comment:
Hm, I see that you reference the other request blocking test page, but I don't understand how the two are different? Why do we need both?

Reply:
Makes sense to me, but I'd defer to @kdzwinel as the privacy test page czar.